src/devices/bus/isa/lbaenhancer.cpp

```diff
@@ -0,0 +1,142 @@
+// license:BSD-3-Clause
+// copyright-holders:Bavarese
+/**********************************************************************
+
+    VS Systems LBA Enhancer (ISA; 1995).
+
+***********************************************************************
+
+    Generic BIOS extension card. Adds 28 bit LBA mode to any pre-1994 
+       board with an empty ISA slot.
+
+    Supports 1 to 4 large hard disks. Bios boot remains possible.
+
+    Check date code of Ami BIOS (at bottom of boot screen):
+    40-0100-001139-00101111-111192-486ABC-F  (111192 will not support LBA)
+    50-0100-001292-00101111-072594-ABCDEF-F  (072594 will support LBA)
+
+    See: ftp://ami.com/archive/utility/LBA.TXT
+
+    Usage:
+      select type 47 for each LBA drive and set C,H,S (params) to '8', 
+      then repartition and reformat (Fdisk and Format). Data is lost.
+
+    Requirements:
+      Compatible OS (DOS >= 6.22) and LBA capable drive(s).
+***********************************************************************
+
+PCB Layout
+----------
+
+|-------------------------------------------|
+|                                           |
+|                                           |
+|                                           |
+|            EPROM     CD74LS245            |
+|                                  J3,J2,J1 |
+|                      PAL16V8              |
+|                                           |
+|                                           |
+|---|                                   |---|
+    |-----------------------------------|
+
+Notes:
+
+J3, J2, J1 set BIOS (start) address to C800, CC00, D000, D400, D800 or DC00
+
+- one CD74LS245 
+- one PALCE16V8H-25
+- one 32 K EPROM : 27C256 (STM) (mirrored after 16 K)
+
+*/
+
+#include "emu.h"
+#include "lbaenhancer.h"
+
+//**************************************************************************
+//  DEVICE DEFINITIONS
+//**************************************************************************
+
+DEFINE_DEVICE_TYPE(LBA_GEN, lba_gen_device, "lba_gen", "VS Systems LBA Enhancer BIOS 1995")
```

Comment:
The device type identifier, class name and short name are far too generic and will cause conflicts.